src/components/outline/AppOutline.tsx

```diff
@@ -10,33 +10,33 @@ import { UserCardType } from "../../types/UserCard";
 const AppOutline: React.FC = () => {
   const [users, setUsers] = useState<any[]>([]);
 
-  // TODO: We need to get the hexathon users from the API
-
-  // useEffect(() => {
-  //   const getUsers = async () => {
-  //     const hexUrl = apiUrl(Service.HEXATHONS, "hexathon-users/647fee51768e521dc8ef88e0/users");
-  //     const hexData = await axios.get(hexUrl);
-  //     console.log(hexData);
-  //     setUsers(data?.data?.profiles);
-  //   };
-  //   getUsers();
-  // }, []);
-
-  // Temporarily resorting to regular users since hackGTX has 0 users
-  // However, the API calls were made above to help
-  const userData: UserCardType[] = [
-    {
-      name: "TEMPORARY USER",
-      profile: {
-        school: "TEMPORARY SCHOOL",
-        year: "TEMPORARY YEAR",
-        major: "TEMPORARY MAJOR",
-        description: "TEMPORARY DESCRIPTION",
-        skills: ["test", "test", "test", "testing again", "another test"],
-        commitmentLevel: "test", //planning on dealing with commitment type and background conditionals
-      }
-    },
-  ];
+  // 647fee51768e521dc8ef88e0 hackGTX
+  // 62d9ed68d0a69b88c06bdfb2 HackGT9
+  // 640680f0c791f79de667dfa9 Test1
+  // 62adf8b633b2c3301a295ed3 Dev1
+
+  useEffect(() => {
+    const getUsers = async () => {
+      const hexUrl = apiUrl(Service.HEXATHONS, "/hexathon-users/647fee51768e521dc8ef88e0/users");
+      const hexData = await axios.get(hexUrl);
+      setUsers(hexData?.data?.hexathonUsers);
+    };
+    getUsers();
+  }, []);
```

Comment:
Nice! but a suggestion, we don't generally use `useEffect` when all it does it hit one API endpoint, which is what we're doing here. We tend to use `useAxios` since it's more intuitive. You can look all over the registration codebase to see examples, here are [two examples](https://github.com/HackGT/registration2/blob/8ce23cb021a3fd5f64b829cfaf76bb0ef251a3da/src/components/admin/applications/ApplicationDetailPage.tsx#L57-L66). After the call, you can have a small check like so: 
```
if (loading) return 'loading...'
```
Later, we can render a nicer loading page, but that's not what this issue is tackling so dwai. The reason we have this check is to wait for the data to be fetched, ie, loading will be set to true until data is fetched, so you won't have to worry about the data returned being undefined. You can look up useAxios docs for more info